proposal/comp110_architecture_proposal.tex

```diff
@@ -18,60 +18,60 @@ \section*{Topic}
+\item[Web link:] \url{http://dl.acm.org/citation.cfm?doid=1401843.1401858}
+\item[Full text link:] \url{https://users.soe.ucsc.edu/~ejw/papers/smith-sandbox-2008.pdf}
+\item[Comments:] This document goes into detail on how to model power ups and how how movement aids can help a player in a level with momentum based game play. The paper has also been cited 14 times.
 \end{description}
 
 \section*{Paper 4}
 \begin{description}
-\item[Title:] Title of paper
-\item[Citation:] \cite{bibtex_key}
-\item[Abstract:] Copy and paste the abstract here
-\item[Web link:] Give the URL of the paper in IEEE Xplore, ACM Digital Library, or similar
-\item[Full text link:] Give the URL of a downloadable PDF of the paper, if you can find one
-\item[Comments:] Write a few sentences on how you found the article and why you believe it is relevant and/or important.
+\item[Title:] Procedural level generation using occupancy-regulated extension
+\item[Citation:] \cite{Mawhorter}
+\item[Abstract:] ``Existing approaches to procedural level generation in 2D platformer games are, with some notable exceptions, procedures designed to do the work of a human game designer. They offer the usual benefits and disadvantages of AI applied to a cognitive task: they can work much faster than a human level designer, and are in some cases able to explore the design space automatically to find levels with desirable qualities. But they aren't able to capture the human creativity that produces the most interesting level designs, and they are usually very specific to their particular domain. This paper introduces occupancy-regulated extension (ORE), a general geometry assembly algorithm that supports human-design-based level authoring at arbitrary scales.''
+\item[Web link:] \url{http://ieeexplore.ieee.org/xpl/articleDetails.jsp?arnumber=5593333&queryText=level\%20generation&newsearch=true}
+\item[Full text link:] \url{https://games.soe.ucsc.edu/sites/default/files/cig10_043CP2_115.pdf}
+\item[Comments:] This paper is great for the discussion on making cohesive randomly generated level designs for a general purpose plat former that fit an aesthetic requirement. The work has also been cited 7 times in the IEEE library.
 \end{description}
 
 \section*{Paper 5}
 \begin{description}
-\item[Title:] Title of paper
-\item[Citation:] \cite{bibtex_key}
-\item[Abstract:] Copy and paste the abstract here
-\item[Web link:] Give the URL of the paper in IEEE Xplore, ACM Digital Library, or similar
-\item[Full text link:] Give the URL of a downloadable PDF of the paper, if you can find one
-\item[Comments:] Write a few sentences on how you found the article and why you believe it is relevant and/or important.
+\item[Title:] The Challenge of Automatic Level Generation for Platform Videogames Based on Stories and Quests
+\item[Citation:] \cite{Mourato}
```

Comment:
Insufficient detail -- could be used as supporting material. Is there a more recent work by Mourato which would be suitable?